src/radio/radio-group.component.ts

```diff
@@ -48,6 +48,7 @@ import { RadioChange } from "./radio-change.class";
 	template: `
 		<div
 			class="bx--radio-button-group"
+			aria-label="radiogroup"
 			[ngClass]="{
```

Comment:
You can do a better job here. You can use `role="radiogroup"` as well as `aria-labelledby` (since the radio group's label should really be whatever piece of text that represents it)

https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html

Reply:
Okay sounds good!